File: cmd/debtdrone/testdata/dirty_code/complex.rb
def complex_function(x)
  if x > 0
    x.times do |i|
      if i % 2 == 0
        if i % 3 == 0
          i.times do |j|
            if j > 5
              puts "Deeply nested"
            end
          end
        end
      end
    end
  else
    if x < -5
      puts "Negative"
    end
  end
end

File: cmd/debtdrone/testdata/dirty_code/complex.swift
func complexFunction(x: Int) {
    if x > 0 {
        for i in 0..<x {
            if i % 2 == 0 {
                if i % 3 == 0 {
                    for j in 0..<i {
                        if j > 5 {
                            print("Deeply nested")
                        }
                    }
                }
            }
        }
    } else {
        if x < -5 {
            print("Negative")
        }
    }
}
